File: 2019 czerwiec/python/2.py
with open("pierwsze.txt", "r") as f:
    liczby = f.read().strip().split('\n')

def is_prime(n):
    if n == 1:
        return False
    
    for i in range(2, n):
        if n % i == 0:
            return False
    return True

with open("wyniki4_2.txt", "w") as f:
    for n in liczby:
        if is_prime(int(n[::-1])):
            f.write(n + '\n')

File: 2019 czerwiec/c++/2.cpp
#include <iostream>
#include <fstream>
#include <algorithm>
#include <string>
using namespace std;

bool is_prime(int n)
{
    if (n <= 1)
        return false;
    for (int i = 2; i * i <= n; i++)
    {
        if (n % i == 0)
            return false;
    }
    return true;
}

int main()
{
    ifstream in("pierwsze.txt");
    ofstream out("wyniki4_2.txt");

    for (int i = 1; i <= 200; i++)
    {
        string a;
        in >> a;
        reverse(a.begin(), a.end());
        if (is_prime(stoi(a)))
        {
            reverse(a.begin(), a.end());
            out << a << '\n';
        }
    }
}